fdbclient/include/fdbclient/ServerKnobs.h

```diff
@@ -578,7 +578,8 @@ class SWIFT_CXX_IMMORTAL_SINGLETON_TYPE ServerKnobs : public KnobsImpl<ServerKno
 	int ROCKSDB_BLOCK_PROTECTION_BYTES_PER_KEY;
 	bool ROCKSDB_METRICS_IN_SIMULATION; // Whether rocksdb traceevent metrics will be emitted in simulation. Note that
```

Comment:
Didn't see a reason to have a separate knob for sharded rocks simulation simulations, but happy to change that if needed.

Reply:
Keeping one knob is sufficient. Usually we just add separate knobs for tuning purpose.